chat_srvc/lib/dbConnect.ts

```
@@ -0,0 +1,58 @@
+ * in development. This prevents connections growing exponentially
+ * during API Route usage.
+ */
+
+interface MongooseCache {
+  conn: typeof mongoose | null;
+  promise: Promise<typeof mongoose> | null;
+}
+
+// Extend the global type to include mongoose cache
+declare global {
+  var mongoose: MongooseCache | undefined;
```

Biome (2.3.13)

[error] Shouldn't redeclare 'mongoose'. Consider to delete it or rename it.

'mongoose' is defined here:


(lint/suspicious/noRedeclare)